windows_containers/configuring-byoh-windows-instance.adoc

```diff
@@ -0,0 +1,65 @@
+[id="configuring-byoh-windows-instance"]
+= Configuring Bring-Your-Own-Host (BYOH) Windows instances
+
```

Comment:
You'll need to add a few things to the header metadata here:
`include::modules/common-attributes.adoc[] `(line 3)
`:context: <configuring-byoh-windows-instance>` (line 4)

`toc::[]` (line 6)